src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { LocalStorageService } from 'src/app/services/local-storage.service';
+import { LoginService } from 'src/app/services/login.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+  pageTitle = 'Food Feedback system';
+  loginForm: FormGroup;
+  submitted = false;
+  Login_User_details: any;
+  get loginform() { 
+    return this.loginForm.controls;
+  }
+  
+  constructor(private formBuilder: FormBuilder ,private loginservice: LoginService, private router: Router, private localstorage: LocalStorageService) { }
+
+  ngOnInit() {
+    this.loginForm = this.formBuilder.group({
+      email_id: ['', Validators.required],
+      password: ['', Validators.required]
+    });
+  }
+  OnLogin() {
+    this.submitted = true;
+    if (this.loginForm.invalid) {
+      return;
+    }
+    this.localstorage.setitem("Email_id", <string>this.loginForm.get("email_id").value);
```

Comment:
It would be great if you can move the logic of copying the emailid to localstorage on successful authentication. Makes sense ?